### PR TITLE
Clarify "Disable" button as an "Enable" Button

### LINKED ISSUE #62: UX Enhancement - "Disable?" should be reversed to "Enable"
Reverse the "Disable?" checkbox to be an "Enable Filters" checkbox.

It is somewhat counter-intuitive to enable a checkbox to disable something.

### PR DESCRIPTION
**Pull request by jnovack**
Reverses text/function of the "Disable Filters" button to be an "Enable Filters" button.   Fixes #62

### CODE REVIEW

**Comment by jnovack** [comment]
Function is fundamentally unchanged, just reversed.

image

File: MainWindow.cs [hunk 1]
```diff
@@ -612,7 +612,7 @@ private async void ButtonAdvanceDate_Click(object sender, EventArgs e)
         private bool CheckAdvanceDate(out bool prompt)
         {
             prompt = false;
-            if (CheckDisable.Checked)
+            if (CheckEnableFilters.Checked)
```
Comment (architdate) [inline MainWindow.cs:615]: This condition needs to be inverted if we are to have checkenabledfilters

**Review by architdate — CHANGES_REQUESTED** [review]
The default should also be set to true (checked)

File: MainWindow.cs (hunk 1 above)
Comment (jnovack) [reply MainWindow.cs]: ....aaaand this is why we do code reviews :)

**Review by architdate — APPROVED** [review]

**Comment by architdate** [comment]
Ty for the edits, merged :)

**Pull request merged**